.mvn/extensions.xml

```diff
@@ -5,7 +5,11 @@
         <artifactId>takari-smart-builder</artifactId>
         <version>0.6.1</version>
     </extension>
-
+    <extension>
+        <groupId>io.takari.aether</groupId>
+        <artifactId>takari-local-repository</artifactId>
+        <version>0.11.3</version>
+    </extension>
```

Comment:
This project is archived and indicates that the maven resolver plugin is the successor, is there a specific reason to use the takari project over the maven resolver plugin?